opennms-doc/guide-admin/src/asciidoc/text/telemetryd/introduction.adoc

```diff
@@ -0,0 +1,40 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+The _Telemetry Daemon_ (_telemetryd_) allows sensors to push metrics from sensors into _{opennms-product-name}_.
```

Comment:
**sensors** to push metrics from **sensors**?

I prefer the previous phrasing: The telemetry daemon (_telemetryd_) provides an extensible framework that can be used to handle sensor data pushed to _{opennms-product-name}_.